src/Extras/autodifferentiation.jl

```diff
@@ -16,6 +16,13 @@ function cumsum(d::DualFun)
     DualFun(cumsum(d.f),(I-Evaluation(rangespace(Q),leftendpoint))*Q)
 end
 
+# total definite integral of a function
+import Base.sum
```

Comment:
This should go in the list of imports in src/ApproxFun.jl (actually I'm pretty sure it's already included)